apple/Elements/RNSVGImage.mm

```diff
@@ -133,9 +143,12 @@ - (void)didReceiveImage:(UIImage *)image metadata:(id)metadata fromObserver:(voi
     // See for more info: T46311063.
     return;
   }
-    auto imageSource = _state->getData().getImageSource();
-    imageSource.size = {image.size.width, image.size.height};
-    static_cast<const RNSVGImageEventEmitter &>(*_eventEmitter).onLoad({imageSource.size.width, imageSource.size.height, imageSource.uri});
+  auto imageSource = _state->getData().getImageSource();
+  imageSource.size = {image.size.width, image.size.height};
+    static_cast<const RNSVGImageEventEmitter &>(*_eventEmitter).onLoad({.source = {
```

Comment:
I think it is nice to add a check for if `_eventEmitter` is nil, same as in RNScreens: https://github.com/software-mansion/react-native-screens/blob/a9bc1118b755c362a981638c0ce0091affd46fe0/ios/RNSScreen.mm#L323.

Reply:
okay, the good point :)

Reply:
done